javagen/src/main/java/com/azure/autorest/model/javamodel/JavaFileContents.java

```diff
@@ -261,6 +261,16 @@ public void javadocComment(int wordWrapWidth, Consumer<JavaJavadocComment> comme
                 commentAction.accept(new JavaJavadocComment(this))));
     }
 
+    void javadocCommentStart() {
+        line("/**");
+        addToPrefix(" * ");
+    }
+
+    void javadocCommentEnd() {
```

Comment:
Why are these package private? Can we make them public if it's used outside this class?

Reply:
I was intended for these as internal usage within javamodel. We do not want dev writing template to call `javadocCommentStart` and `javadocCommentEnd` explicitly, as it is possible to mismatch the start/end call.

Though no strong opinion on this. Anyway only our dev uses it.